### DISCUSSION TITLE
Consider showing branch count

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44019] [Mobile] [answered]
Visit community/events
![Screenshot_20230111-170214.png](https://user-images.githubusercontent.com/2119212/211927415-1f658cf8-7a2e-41fd-83b2-158bfa80ff60.png)
Is it worth tapping change branch?
![Screenshot_20230111-170222.png](https://user-images.githubusercontent.com/2119212/211927489-cea9da16-d608-4e1d-84b8-dc017ee3a3f0.png)
Nope.

Worst case... Visit Microsoft/PowerToys (possibly on some random long branch):
![Screenshot_20230111-170151.png](https://user-images.githubusercontent.com/2119212/211927672-75b68c99-b538-4f02-95e0-244fa0e76d16.png)
Is it worth changing? You won't know until you tap
![Screenshot_20230111-170133.png](https://user-images.githubusercontent.com/2119212/211927780-c0d2f8d3-7905-47a2-8db9-c49793101e95.png)
If the branch count was displayed on the screen near the change branch, I'd know without having to tap.

On desktop it's generally shown unless the window is very narrow.

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref thanks for the feedback. I passed this on to the team for consideration.
